src/main/java/pl/agawesolowska/ticketbookingapp/service/ScreeningService.java

```diff
@@ -0,0 +1,48 @@
+package pl.agawesolowska.ticketbookingapp.service;
+
+import java.time.LocalDateTime;
+import java.util.Optional;
+
+import javax.transaction.Transactional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import pl.agawesolowska.ticketbookingapp.dao.ScreeningRepository;
+import pl.agawesolowska.ticketbookingapp.model.entity.Screening;
+
+/**
+ * @author Aga Wesołowska
+ *
+ */
+@Service
+public class ScreeningService {
+
+	private final ScreeningRepository screeningRepository;
+
+	@Autowired
+	public ScreeningService(ScreeningRepository screeningRepository) {
+		this.screeningRepository = screeningRepository;
+	}
+
+	@Transactional
+	public Screening getScreeningById(Long id) {
+		Optional<Screening> optionalScreening = screeningRepository.findById(id);
+		if (optionalScreening.isEmpty()) {
```

Comment:
dlaczego nie wykorzystać potencjału Optionala do zrobienia tego w lepszym stylu?

Reply:
Takie coś będzie na pewno lepsze:
`return screeningRepository.findById(id).orElseThrow(() -> new IllegalArgumentException("The screening ID is invalid."));`